### DISCUSSION TITLE
Animated progress bars

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/stylesheet#935] [Ideas]
Something I've noticed when installing large Flatpaks or copying large files over the network: sometimes it's hard to tell if the progress bar is still "doing" anything; when the progress bar isn't noticeably changing because a process is taking a long time, it leads me to think it might have stalled or even frozen up. I wonder if we could add a subtle animation to the progress bar to help reassure users that the process is still ongoing and things aren't frozen.

## Prior Art

Windows uses a "filling" animation on determinate progress bars. Material Design has an animation for indeterminate progress, but does not have an animation on determinate progress. I am unsure about macOS and iOS.

**Comment by danirabbit** [comment]
I'm not sure how you close a discussion but this is done now :)

**Reply by cassidyjames** [reply]
Implemented in https://github.com/elementary/stylesheet/pull/937
